sdk/storage/azure-storage-blob/azure/storage/blob/aio/_download_async.py

```diff
@@ -111,14 +109,8 @@ async def _download_chunk(self, chunk_start, chunk_end):
                         **self.request_options
                     )
                     retry_active = False
-
                 except HttpResponseError as error:
                     process_storage_error(error)
-                except ClientPayloadError as error:
```

Comment:
`AioHttpTransport` will catch `ClientPayloadError` and raise it as a `ServiceResponseError` which inherits from `HttpsResponseError`.